api/observability/v1/output_types.go

```diff
@@ -821,30 +821,69 @@ type Syslog struct {
 	// +kubebuilder:default:=user
 	Facility string `json:"facility,omitempty"`
 
-	// PayloadKey specifies record field to use as payload.
+	// PayloadKey specifies record field to use as payload. This supports template syntax
```

Comment:
1. Do we need to mention the "default" key is "message" and if you specify nothing then that is what is used?
2. Does this one need special restriction to only allow you to use the "default" or a path to a field?  This seems to be one field where a static value or combination doesn't make any sense

Reply:
I've refactored to allow field paths encased in brackets. It will default to `.message` when it is left blank